gpflux/optimization/keras_natgrad.py

```diff
@@ -36,18 +36,42 @@
 
 
 class NatGradModel(tf.keras.Model):
-    """
+    r"""
     This is a drop-in replacement for `tf.keras.Model` when constructing GPflux
     models using the functional Keras style, to make it work with the
     NaturalGradient optimizers for q(u) distributions in GP layers.
 
+    You must set the `natgrad_layers` property before compiling the model. Set it
+    to the list of all :class:`~gpflux.layers.GPLayer`\ s you want to train using
+    natural gradients. You can also set it to `True` to include all of them.
+
     This model's :meth:`compile` method has to be passed a list of optimizers, which
-    must be one `gpflow.optimizers.NaturalGradient` instance per
+    must be one `gpflow.optimizers.NaturalGradient` instance per natgrad-trained
     :class:`~gpflux.layers.GPLayer`, followed by a regular optimizer (e.g.
     `tf.keras.optimizers.Adam`) as the last element to handle all other
     parameters (hyperparameters, inducing point locations).
     """
 
+    @property
+    def natgrad_layers(self) -> List[GPLayer]:
```

Comment:
```suggestion
    def natgrad_layers(self) -> List[GPLayer]:
        """
        The list of layers in this model that should be optimized using `~gpflow.optimizers.NaturalGradient`.
        
        :getter: Returns a list of the layers that should be trained using `~gpflow.optimizers.NaturalGradient`.
        :setter: Sets the layers that should be trained using `~gpflow.optimizers.NaturalGradient`.
            Can be an explicit list or a `bool`: If set to `True`, it will select all `GPLayer` instances in the model layers.
        """
```